src/ros_tcp_endpoint/tcp_sender.py

```diff
@@ -55,10 +55,10 @@ def send_unity_message(self, topic, message):
 
         try:
             s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            s.settimeout(2)
+            s.settimeout(10)
```

Comment:
This value should be configurable and we should be able catch when the sendall command times out and surface that to the user.